crm_location_nuts/tests/test_crm_location_nuts.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
```

Comment:
And when you are at it anyway: change the license lines to https:

Reply:
Did it, thank you